src/main/java/org/commcare/formplayer/application/FormplayerSentryFilter.java

```diff
@@ -15,7 +14,7 @@
 import java.io.IOException;
 
 @Component
-@Order(Ordered.HIGHEST_PRECEDENCE + 2)
+@Order(1)
```

Comment:
This is set to the same value as the RequestResponseLoggingFilter below, is that intentional?

Even if we no longer need strict ordering, don't we still not want to be enforcing an order on these two filters for consistency?

Reply:
We could chose and arbitrary order, any suggestions on which should go first?

Reply:
decided to put Sentry first: c31bbf10

Reply:
Logging first makes sense to me. thanks!